forte/data/ontology/top.py

```diff
@@ -1168,12 +1170,74 @@ def __init__(
         )
 
 
+class Payload(Entry):
+    def __init__(
+        self,
+        pack: PackType,
+        modality: str,
+        payload_idx: int,
+    ):
+        self.payload_idx = payload_idx
+        self.modality = modality
+        super().__init__(pack)
+        self.cache = []
+        self.meta = {}
+
+    def load(self, path):
+        data, data_meta = self.loading_method(path)
+        self.cache.append(data)
+        self.meta = {**self.meta, **data_meta}
```

Comment:
This wildcard creation basically means we don't have any control here, let's reduce such patterns